### DISCUSSION TITLE
[Fixed] Artboards Sort Order

### DISCUSSION THREAD

**Discussion by cargeo** [ingrammicro/puzzle-publisher#64] [General]
Hello,

if you set ‚Artboards Sort Order“ to „left-to-right“, all vertical arranged artboards get ordered from bottom up. It should be top to bottom. Is this possible?

See screenshot for my scenario:
https://imgur.com/a/diRJGSl

**Comment by MaxBazarov** [comment]
@cargeo  It should work in the way you described. On my sample it works fine. Could you send your example to mbazarov@gmail.com to allow me to reproduce.

**Comment by MaxBazarov** [comment]
Fixed the issue in 16.11.2 update.

**Comment by cargeo** [comment]
Thank you!
